eth/executionclient/execution_client.go

```diff
@@ -250,7 +262,12 @@ func (ec *ExecutionClient) Healthy(ctx context.Context) error {
 	if sp != nil {
 		ec.logger.Error("Execution client is not synced")
 		ec.metrics.ExecutionClientSyncing()
-		return fmt.Errorf("syncing")
+		syncDistance := max(sp.HighestBlock, sp.CurrentBlock) - sp.CurrentBlock
+
+		// block out of sync distance tolerance
+		if syncDistance > ec.syncDistanceTolerance {
+			return fmt.Errorf("sync distance exceeds tolerance (%d): %w", syncDistance, errSyncing)
+		}
```

Comment:
The log `ec.logger.Error("Execution client is not synced")`
should be moved inside the `if syncDistance > ec.syncDistanceTolerance` block
to avoid unnecessary logging when the sync distance is within tolerance.

Reply:
https://github.com/ssvlabs/ssv/issues/1960

Reply:
> #1960

there's a follow up PR in which this will be fixed.